Sources/MySQL/Internals/Handshake.swift

```diff
@@ -46,7 +46,12 @@ extension Packet {
         
         // Require decimal `10` to be the protocol version
         guard try parser.byte() == 10 else {
-            throw MySQLError(.invalidHandshake)
+            // if the first byte is 0xff then we got an error packet
+            if(parser.payload[0] == 0xff){
```

Comment:
Code style: To match the style found elsewhere in this file, write this line as `if parser.payload[0] == 0xff {`